django_emarsys/migrations/0003_newevent.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_emarsys', '0002_neweventinstance'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='NewEvent',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('name', models.CharField(unique=True, max_length=1024)),
+                ('emarsys_id', models.IntegerField()),
+            ],
+            options={
+                'ordering': ['name'],
+                'permissions': [('can_trigger_event', 'Can trigger emarsys events.')],
+            },
+            bases=(models.Model,),
+        ),
+    ]
```

Comment:
Could you merge migrations such that we have as little new migrations as possible? I believe that makes migrating faster.